diesel_cli/README.md

```diff
@@ -4,6 +4,15 @@ Diesel CLI
 Diesel CLI is a tool that aids in managing your database schema. Migrations are
 bi-directional changes to your database that get applied sequentially.
 
+Installation
+---------------
+
+The diesel cli requires `mysql` to be installed on your system. If you have
+homebrew installed, you can run `which mysql || brew install mysql` to install
+`mysql` if necessary.
```

Comment:
ah, the reason I suggested brew is that the `mysql` website doesn't really help at all :-D the "install for mac os x" page, for example, references a DMG but provides no links to download it...